components/BottomNavigation/tests/unit/BottomNavigationTests.m

```diff
@@ -0,0 +1,85 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialBottomNavigation.h"
+#import "../../src/private/MDCBottomNavigationItemView.h"
```

Comment:
This doesn’t seem ideal. Are you unable to reference the header using private/ as a prefix alone?

Reply:
This is the structure we've chosen.  I didn't want to depend on the INCLUDE search path to find the correct file.

```
src/*
src/private/*
tests/unit/*
tests/earlgrey/*
examples/*
examples/supplemental/*
```

Reply:
We are already depending on the include search path for our umbrella headers. I’m not sure what you mean by “this is the structure we’ve chosen” here.

This is the first example I’m aware of that makes use of relative import paths like this - are you sure we want to start encouraging this?

Reply:
Xcode fails to find the file if I do

```objectivec
#import "private/MDCBottomNavigationItemView.h"
```

```
/Users/.../BottomNavigationTests.m:20:9: fatal error: 'private/MDCBottomNavigationItemView.h' file not found
#import "private/MDCBottomNavigationItemView.h"
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

```